jenkins/scripts/cbts/coverage_utils/sitecustomize.py

```
@@ -0,0 +1,206 @@
+                try:
+                    with open(_MARKER_FILE) as f:
+                        nodeid = f.read().strip()
+                    if nodeid and nodeid != last_seen:
+                        cov.switch_context(nodeid)
+                        try:
+                            # Save immediately so short-lived worker
+                            # processes (SIGKILL'd before next periodic
+                            # save) still leave their context+data.
+                            cov.save()
+                        except Exception:
+                            pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)